src/bin/bat/app.rs

```diff
@@ -207,7 +207,10 @@ impl App {
                 || match self.matches.get_one::<String>("color").map(|s| s.as_str()) {
                     Some("always") => true,
                     Some("never") => false,
-                    Some("auto") => env::var_os("NO_COLOR").is_none() && self.interactive_output,
```

Comment:
Thanks! Would be very nice if you could add a helper function to src/bin/bat so we don't have to duplicate this logic 🙏